project_remaining_time_on_task/models/project.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from openerp import api, fields, models
+
+
+class Task(models.Model):
+    _inherit = 'project.task'
+
+    has_contract_hours = fields.Boolean(string="Has contract hours",
+                                        compute="_compute_has_contract_hours")
+
+    remaining_contract_hours = fields.Float(
+        string="Remaining Contract Hours",
+        compute='_compute_remaining_hours_contract',
+        digits=(16, 2))
+
+    def _get_qty_invoiced(self):
+        _qty_invoiced = 0
+        sale_order = self.env['sale.order'].search(
+            [('project_id', '=', self.analytic_account_id.id)])
+        for sale_order_lines in sale_order.order_line:
```

Comment:
It is working here because sale_order_lines is a recordset of 1 but, if I'm not mistaken, it should rather be:

```
qty_invoiced = sum(line.qty_invoiced for line in sale_order.order_line)
```

Reply:
@nicolasj77